src/pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -6,7 +6,10 @@
 
   <PropertyGroup>
     <Version>$(SharedFrameworkNugetVersion)</Version>
```

Comment:
Actually, I think this is one of the few correct usages of this property.  `Microsoft.NETCore.App`'s version is `$(SharedFrameworkNugetVersion)`. It just so happens they always align today.

If we ever were servicing something like UWP and wanted to keep `Microsoft.NETCore.App` static, like we do for Host and HostFxr, this should be ProductVersion.

All that being said, I don't know if we'd ever want/need that.  My guidance is:  if you are generally talking about the version, not tied to any specific component, use `ProductVersion`.  If you are specifically referencing a component, and it has its own version property, use it.